tests/test_mtdome_csc.py

```diff
@@ -653,5 +691,87 @@ async def test_status_error(self):
                 faultCode=expected_fault_code,
             )
 
+    async def test_exitFault(self):
+        async with self.make_csc(
+            initial_state=salobj.State.STANDBY,
+            config_dir=None,
+            simulation_mode=1,
+        ):
+            await self.set_csc_to_enabled()
+
+            # Prepare the lower level components
+            self.csc.mock_ctrl.amcs.status = MotionState.FAULT
+            self.csc.mock_ctrl.apscs.status = MotionState.FAULT
+            self.csc.mock_ctrl.lcs.status[:] = MotionState.FAULT.name
+            self.csc.mock_ctrl.lwscs.status = MotionState.FAULT
+            self.csc.mock_ctrl.moncs.status = MotionState.FAULT
+            self.csc.mock_ctrl.thcs.status = MotionState.FAULT
+            self.csc.mock_ctrl.amcs._commanded_motion_state = MotionState.FAULT
+            self.csc.mock_ctrl.lwscs._commanded_motion_state = MotionState.FAULT
+
+            await self.csc.write_then_read_reply(command="exitFault")
+
+            await self.csc.statusAMCS()
+            amcs_status = self.csc.lower_level_status[LlcName.AMCS.value]
```

Comment:
Is `.value` necessary here? I think the enum class is a subclass of enum.IntEnum so I would expect it not to be needed.

Reply:
The statuses are passed on as strings, so it is necessary.

Reply:
Ah, I see that LlcName is an enum.Enum, not an enum.IntEnum. If you made it inherit from str as well as enum.Enum then you could skip the `.value` but it's not a big deal.